ios/sdk/src/JitsiMeet.m

```diff
@@ -214,12 +230,20 @@ - (NSDictionary *)getDefaultProps {
     return _defaultConferenceOptions == nil ? @{} : [_defaultConferenceOptions asProps];
 }
 
+- (RCTBridgeWrapper *)bridgeWrapper {
+    if (_bridgeWrapper == NULL) {
+        _bridgeWrapper = [[RCTBridgeWrapper alloc] init];
```

Comment:
This feels wrong, we shouldn't be creating one here. What are we trying to solve with this?

Reply:
this is how lazy initialization is done in obj c